artifacts/attestation-authority-example.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kritis.grafeas.io/v1beta1
```

Comment:
Maybe move this to artifacts/examples/ ?

Reply:
done